recipes/pcre/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
+
+from conans import ConanFile, CMake, tools
+import os
+
+
+class PCREConan(ConanFile):
+    name = "pcre"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.pcre.org"
+    author = "Bincrafters <bincrafters@gmail.com>"
+    description = "Perl Compatible Regular Expressions"
+    license = "BSD"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```
I've read the pcre's license file. There is a mention about BSD, but reading all license file you can file all clauses from version 3 there.